build-data-capturing-gradle-samples/capture-thermal-throttling/gradle-thermal-throttling.gradle

```diff
@@ -30,12 +30,17 @@ if (OperatingSystem.current().macOsX) {
   // Register Thermal throttling service
   def throttlingServiceProvider = gradle.sharedServices.registerIfAbsent("thermalThrottling", ThermalThrottlingService.class) {}
 
-  // Start Thermal throttling service
-  throttlingServiceProvider.get()
+  buildScan.background {
```

Comment:
@jprinet This seems odd that we need to do this. Is there a conversation with the configuration cache team about this?

If not, could you follow up with the configuration cache team to see if there is a better way to do this or a bug must be filed?